Fix endianness detection: endian.h should be present on all Debian
machines. This patch forces the use of endian.h, this preventing
several reverse dependencies os Numpy from failing to build.

Index: python-numpy-1.4.1/numpy/core/include/numpy/npy_endian.h
===================================================================
--- python-numpy-1.4.1.orig/numpy/core/include/numpy/npy_endian.h	2010-04-22 11:35:23.000000000 +0200
+++ python-numpy-1.4.1/numpy/core/include/numpy/npy_endian.h	2010-06-03 10:52:58.000000000 +0200
@@ -6,7 +6,8 @@
  * endian.h
  */
 
-#ifdef NPY_HAVE_ENDIAN_H
+/*#ifdef NPY_HAVE_ENDIAN_H*/
+#if 1
     /* Use endian.h if available */
     #include <endian.h>
 
